src/migrations/20220427130214-add-sats-amount-prop-to-payments.ts

```diff
@@ -0,0 +1,99 @@
+module.exports = {
+  async up(db) {
+    const collection = db.collection("medici_transactions")
+    try {
+      const resultDebits = await collection.update(
+        { type: "payment", debit: { $gt: 0 } },
```

Comment:
if we only include payments the db will remain inconsistent

Reply:
The migration/new-code covers all transactions that are related to the `hash` (`payment` and `fee_reimbursements`).

I'm planning to migrate the rest of transaction types and redo the domain logic in separate PRs. The scope here is just not to lose info when we start doing USD payments. It's the bare minimum to get #1237 merged in.